docs/_asset/index.css

```diff
@@ -789,11 +791,54 @@ button.success {
   );
 }
 
+.react-tabs__tab-panel {
+}
+
+.react-tabs__tab-panel--selected {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+/* Preview / Options */
+.react-tabs__tab-panel--selected.tab-panel-scrollable {
+  overflow: auto;
+}
+
 .frame-body {
   border-bottom-left-radius: 3px;
   border-bottom-right-radius: 3px;
 }
 
+.react-tabs__tab-panel--selected > .frame-body {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.react-tabs__tab-panel--selected:not(.tab-panel-scrollable) > .frame-body {
+  overflow: hidden;
+}
+
+.frame-body > div,
```

Comment:
This selector applies to the default “Box” (`padding: 1rem; background-color: violet;`) in the example, making it fill the available width. Is that intentional?

Reply:
You mean the available height? 

Not intentional, fixed with better CSS scoping: `.frame-body > .playground-editor-code-mirror`